### PR TITLE
Added cluster provisioning job and capabilities to the provisioning.py

### PR DESCRIPTION
**Pull request by BaptisteGerondeau**
machine_list is in the form of a string with elements separated by commas.

### CODE REVIEW

**Comment by BaptisteGerondeau** [comment]
machine_mrp_provision jobs number 15 (for cluster) and 17 (for individual machine) prove that this is working.

File: files/provisioning.py [hunk 1]
```diff
@@ -5,10 +5,10 @@
 
 
 class ProvisioningJobMaker(object):
-    def __init__(self, working_dirp, machine_name, provisioner_url,
+    def __init__(self, working_dirp, machine_list, provisioner_url,
                  provisioner_token):
         self.working_dir = working_dirp
-        self.machine = machine_name
+        self.machine_list = machine_list.split(', ')
```
[the remark this answers is not in the document]
Comment (BaptisteGerondeau) [reply files/provisioning.py]: split will not trim spaces.
"hpc-blabla, hpc.blabla".split(',') = ['hpcblabla', ' hpc-blabla'] != ['hpc-blabla', 'hpc-blabla']

[the remark this answers is not in the document]
Comment (rengolin) [reply files/provisioning.py]: Yup, read that wrong. Still, the split is by comma, we can trim spaces later when we use the elements to create filenames.

File: files/cluster_provision.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                                 - run-update
+                                 - scm-tag
+                         hpc-sig-devel:
+                                 - job-build
+                                 - job-read
+         parameters:
+                 - choice:
+                         name: machine_type
+                         choices:
+                                 - d05
```
Comment (rengolin) [inline files/cluster_provision.yml]: also, let's use the names we have in the file server: "huawei" and "qualcomm" for these.

**Pull request merged**